SPECS/mariner-rpm-macros/macros

```diff
@@ -74,7 +74,7 @@
     %{_rpmconfigdir}/brp-strip-comment-note %{__strip} %{__objdump} \
     %{_rpmconfigdir}/brp-strip-static-archive %{__strip} \
     %{?py_auto_byte_compile:%{?__brp_python_bytecompile}} \
-    find %{buildroot} -name "*.pc" | xargs -I{} sed -i -e 's@-Wl,-dT,%{_topdir}/BUILD/module_info.ld@ @' {} \
+    find %{buildroot} -name "*.pc" | xargs -I{} sed -i -e "s@-Wl,-dT,%{_topdir}/BUILD/module_info.ld@ @" {} \
```

Comment:
All of our macro files seem to use single quotes for `sed` and the macro you're modifying right now also works without issues in our official, buddy, and dev builds, so I'd prefer to leave it as it is, to avoid hitting some unpredictable build issues.

It might be worth checking what shell is `/bin/sh` pointing to on your build system - that's what `rpmbuild` will use and it's possible, that this or other local system configuration influences your build.